public/app/plugins/datasource/influxdb/migrations.ts

```diff
@@ -0,0 +1,42 @@
+type LegacyAnnotation = {
+  query?: string; // exists in the tag but duplicated above
+  queryType?: string;
+  fromAnnotations?: boolean;
+  tagsColumn?: string;
+  textColumn?: string;
+  timeEndColumn?: string;
+  titleColumn?: string;
+  name?: string;
+};
+
+// {
+//   "datasource": {
+//       "type": "influxdb",
+//       "uid": "P8E9168127D59652D"
+//   },
+//   "enable": true,
+//   "iconColor": "red",
+//   "name": "influxQL",
+//   "query": "SELECT \"usage_idle\" FROM \"cpu\" WHERE (\"cpu\" = 'cpu-total') AND (\"usage_idle\" < 85) AND $timeFilter"
+// }
+
+// this becomes the target in the migrated annotations
+const migrateLegacyAnnotation = (json: LegacyAnnotation) => {
+  return {
+    query: json.query || '', // exists in the tag but duplicated above
+    queryType: 'tags',
+    fromAnnotations: true,
+    tagsColumn: json.tagsColumn || '',
+    textColumn: json.textColumn || '',
+    timeEndColumn: json.timeEndColumn || '',
+    titleColumn: json.titleColumn || '',
+    name: json.name || '',
+  };
+};
+
+//
+export const prepareAnnotation = (json: any) => {
+  json.target = json.target ?? migrateLegacyAnnotation(json);
+
+  return json;
+};
```

Comment:
If you feel the need to use `any`, please use a `// eslint-ignore-next-line` comment instead to allow it, rather than add it to betterer.

Betterer is only a temporary tool to grant amnesty to existing issues, with the goal for it to contain no issues. For _new_ code that we're comfortable violating the lint rules, please add the `// eslint-ignore-next-line` comment 😌